guides/common/modules/proc_installing-the-sos-package.adoc

```diff
@@ -1,14 +1,15 @@
 [id="installing-the-sos-package_{context}"]
-= Installing the SOS Package on the Base Operating System
+= Installing the sos package on the base operating system
 
-Install the *sos* package on the base operating system so that you can collect configuration and diagnostic information from a {EL} system.
 ifdef::satellite[]
+Install the `sos` package on the base operating system so that you can collect configuration and diagnostic information from a {RHEL} system.
```

Comment:
```suggestion
Install the `sos` package on the base operating system so that you can collect configuration and diagnostic information from a {EL} system.
ifdef::satellite[]
```